NewSpring/ApollosProject
# (DEV-3687) Wire Up Add Prayer Form

## DESCRIPTION

### What does this PR do, or why is it needed?

This adds the "Add a Prayer" card and form to the Prayer tab.

### How do I test this PR?

Boot up the app, click around in "Add a prayer", make sure the form works and it doesn't get stuck anywhere trying to navigate back and forth.

---

> I am affirming this is my _best_ work ([Ecclesiastes 9:10](https://www.bible.com/bible/97/ECC.9.10.MSG))

## TODO

- [x] Closes [DEV-3687](https://newspring.atlassian.net/browse/DEV-3687)

## REVIEW

**Manual QA**


**Code Review: Questions to consider**


---

> The purpose of PR Review is to _improve the quality of the software._
